langserver/config.go

```diff
@@ -10,3 +10,26 @@ var (
 	// worth it.
 	envWarmupOnInitialize = os.Getenv("GOLSP_WARMUP_ON_INITIALIZE")
 )
+
+type Config struct {
+	// FuncSnippetEnabled enables the returning of enable argument snippets
+	// on `func` completions, eg. func(foo string, arg2 bar).
+	// Requires code completion to be enabled.
+	FuncSnippetEnabled bool
+	// GocodeCompletionEnabled enables code completion feature (using gocode)
+	GocodeCompletionEnabled bool
+	// MaxParallelism controls the maximum number of goroutines that should be used
+	// to fulfill requests. This is useful in editor environments where users do
+	// not want results ASAP, but rather just semi quickly without eating all of
+	// their CPU.
+	MaxParallelism int
```

Comment:
`MaxParallelism` doesn't seem to be used anywhere.  Should it be removed?

Reply:
Nice catch. Yes it can be removed.

Reply:
Well... It should have been used.

https://github.com/sourcegraph/go-langserver/blob/a28dd2535cb72e3748c9b0292565f26a25046be9/langserver/symbol.go#L343-L351

Notice the magic number `8` in L351. Look like it might originally meant to be `MaxParallelism`. But I am not sure if such customization is still needed.

Reply:
Let me know which way you'd like to go and I'll keep or remove the config var.

Reply:
hah. Ok lets use it at this point.